src/main/java/xyz/nucleoid/plasmid/world/bubble/HasBubbleWorld.java

```diff
@@ -0,0 +1,10 @@
+package xyz.nucleoid.plasmid.world.bubble;
+
+import javax.annotation.Nullable;
+
+public interface HasBubbleWorld {
```

Comment:
hmmm maybe this should be named `BubbleWorldHolder` or something? the "Has" prefix seems a bit weird to me